diff --git a/build/read-commands.js b/build/read-commands.js
index 3f4dacbc99e171001fcc2298372a2f2d559f0ab1..8ab6c705f8d697ae3aca20fc1a5ecec67d178f5a 100644
--- a/build/read-commands.js
+++ b/build/read-commands.js
@@ -1,6 +1,8 @@
 import fs from 'node:fs/promises';
 import path from 'node:path';
 import decamelize from 'decamelize';
+import { pathToFileURL } from 'node:url'
+
 const readCommands = async (directory) => {
     const commands = new Map();
     const files = await fs.readdir(directory);
@@ -31,7 +33,8 @@ const readCommands = async (directory) => {
         if (!/\.(js|ts)x?$/.test(file) || file.endsWith('.d.ts')) {
             continue;
         }
-        const m = (await import(filePath));
+        const fileUrl = pathToFileURL(filePath);
+        const m = (await import(fileUrl));
         const name = decamelize(file.replace(/\.(js|ts)x?$/, ''), { separator: '-' });
         commands.set(name, {
             name,